components/LinkList/src/LinkList.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import clsx from 'clsx';
+
+export interface LinkListProps {
```

Comment:
Ik denk dat het een goed idee is om de LinkListProps te laten extenden van HTMLAttributes<HTMLUListElement>. Dan kan het component net wat flexibeler ingezet worden indien nodig.